### ISSUE TITLE
MathJax rendering error

### ISSUE THREAD

**Issue by dtu-compute** [dtu-compute/course-website#47]
If you look at https://01005-staging.compute.dtu.dk/OpgFU12LD4

then some math symbols are below the lines and there is a `[Math Processing Error]`.

Clicking Update in Admin->StackEdit fixes the problem until the container is rebuild.

**Comment by iainbryson** [comment]
I'm not certain how to reproduce this.  I rebuilt the container but the file seems correct, though there is begin/end:question/answer that I don't think we've discussed implementing.

**Comment by dtu-compute** [comment]
Very strange. I get it every time I login. Also with Chrome. Here is what it looks like for me. Notice that e.g. the omega sign is below the line.

![1](https://cloud.githubusercontent.com/assets/13518486/18050171/3c14c704-6ded-11e6-97ca-8162f5e3ff16.png)

You don't get this in your browser?

I'll have to follow up on begin/end:question/answer thing =)

**Comment by iainbryson** [comment]
Hm. No, I don't see it in my browsers.  This might be trouble to debug...

According to [this](http://meta.math.stackexchange.com/questions/3627/why-is-math-processing-error-all-over-the-place-today), maybe a hard page refresh will fix it.  

There are also indications [here](https://github.com/mathjax/MathJax/issues/1032) that it can be caused by messed up caching, or by bugs in mathjax itself.  Is there any indication from the webconsole of what's going wrong?

Which OS/browser combinations do you use?  I've tested on both FF (48) and Chrome (52.0.2743.116) on OSX, and it renders nicely:

![image](https://cloud.githubusercontent.com/assets/1531103/18060005/b28e5818-6dd0-11e6-917b-262370007d36.png)

**Comment by dtu-compute** [comment]
hmm. In firefox-48.0.1-1.fc23.x86_64 on Fedora 23 at home it rendered wrong, then I did a regular reload and it got fixed, and I can't reproduce it again right now.

Let me try again tomorrow at work where I have reset/cleared the cache.

**Comment by dtu-compute** [comment]
I am closing as it looked like an browser caching issue like you said.
